examples/dt/compute/compute-starter-kit/kustomization.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+  # currently the lib versions is hard coded to assume 3 node openshift
+  # this dt example is for a single node openshift so we need to vendor
+  # the nncp lib and override the nodes, this is a temporary workaround
+  # until the lib is updated to support single node openshift.
+  # - ../../../../lib/nncp
+components:
+- vendor/nncp
+- ../../../../lib/networking
+- ../../../../lib/control-plane
+
+resources:
+- values.yaml
+
+# remove all the openstack components except for keystone, glance, nova
+# neutron, placement, and  deploy only the inra services needed to support
+# the deployed openstack services.
+
+patches:
+- patch: |-
+    - op: remove
+      path: /spec/ironic
+    - op: remove
+      path: /spec/ceilometer
+    - op: remove
+      path: /spec/cinder
+    - op: remove
+      path: /spec/horizon
+    - op: remove
+      path: /spec/heat
+    - op: remove
+      path: /spec/dns
+    - op: remove
+      path: /spec/manila
+    - op: remove
+      path: /spec/mariadb
+    - op: remove
+      path: /spec/octavia
+    - op: remove
+      path: /spec/redis
+    - op: remove
+      path: /spec/swift
+    # - op: remove
+    #   path: /spec/barbican
+  target:
+    kind: OpenStackControlPlane
+    name: controlplane
+    version: v1beta1
+- path: control-plane/disable.yaml
+- path: control-plane/infra.yaml
+- path: control-plane/nova.yaml
```

Comment:
glance and neutron seam to be defaulting to 0 replicas so i just need to add a patch file for each and they should deploy